Env/CarriersList.php

```diff
@@ -57,8 +57,8 @@ class Env_CarriersList extends Env_WebService
 	 */
 	public function getCarriersList($channel, $version)
 	{
-		$this->param['channel'] = Tools::strtolower($channel);
-		$this->param['version'] = Tools::strtolower($version);
+		$this->param['channel'] = strtolower($channel);
+		$this->param['version'] = strtolower($version);

```

Comment:
We want to avoid any module's code in our library (all the files in the Env folder) but we can still add it back if this is a problem for you